### PR TITLE
feature/MBA-30-create-card-component

### PR DESCRIPTION
**Pull request by stepan103ua**

### CODE REVIEW

**Review by Xaiodo — APPROVED** [review]
LGTM

File: src/components/Card/MoneyText/MoneyText.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import styles from './moneyText.styles';
+import { IMoneyTextProps } from './moneyText.types';
+import { currencies, maxNumberOfLines } from './moneyText.settings';
+import { numberWithCommas } from './moneyText.helper';
+
+const MoneyText: FC<IMoneyTextProps> = ({ money, currency }) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text} numberOfLines={maxNumberOfLines}>{`${
```
Comment (milkanuch) [inline src/components/Card/MoneyText/MoneyText.tsx]: put it in the variable

File: src/components/Card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  onIconPress,
+  money,
+  currency,
+}) => {
+  const [isSecured, setSecured] = useState(true);
+  const getStyles = styles(widthScale);
+  const handleLongPress = () => {
+    setSecured(!isSecured);
+    copyTextToClipboard(cardNumber);
+    showSnackBar('The card number has been copied to the clipboard');
```
Comment (milkanuch) [inline src/components/Card/Card.tsx]: put it in the settings variable

**Review by milkanuch — CHANGES_REQUESTED** [review]

**Review by milkanuch — APPROVED** [review]
Now Good!

**Pull request merged**